tests/test-values.yaml

```diff
@@ -5,3 +5,6 @@ image:
 crd:
   ciliumNetworkPolicy:
     enabled: false
+
```

Comment:
Both Cilium and ServiceMonitor CRDs can be used with app-test-suite 0.4.1 (See https://github.com/giantswarm/cert-manager-app/blob/main/.circleci/config.yml#L78 for example circleCI config)

You may need to update python test dependencies manually

Reply:
nice nice nice. I'll prepare another PR with that, thanks!